e2e/cypress/integration/messaging/no_matches_for_autocomplete_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```

Comment:
This is fine to leave as is since the tests could be stable enough to be considered as prod. But in the future, let's start without it, test in CI for a while, and then promote whenever observed to be stable.